lib/generators/rolemodel/linters/eslint/templates/eslint.config.js

```diff
@@ -0,0 +1,98 @@
+import globals from 'globals'
+
+export default [
+  {
+    ignores: ['app/assets/builds/**/*']
+  },
+  { name: 'airbnb' },
+  {
+    languageOptions: {
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true
+        },
+        ecmaVersion: 'latest',
+        globals: {
+          ...globals.browser,
+          ...globals.es6
+        }
+      }
+    },
+    rules: {
+      'no-unused-vars': [
+        'error',
+        {
+          argsIgnorePattern: '^_', // Allow unused arguments starting with an underscore
+          varsIgnorePattern: '^_' // Allow unused variables starting with an underscore
+        }
+      ],
+
+      // No semicolons
+      semi: ['error', 'never'],
+      'no-unexpected-multiline': 'error',
+
+      // 1. We follow the convention of naming "private" properties with a leading underscore
+      // 2. This conflicts with the no-unused-vars config
+      'no-underscore-dangle': 'off',
+
+      // Don't need trailing commas everywhere
+      'comma-dangle': ['error', 'never'],
+
+      // Multiple variable declarations (without assignment) on the same line
+      // without multiple let/const
+      'one-var': ['error', { initialized: 'never', uninitialized: 'always' }],
+      'one-var-declaration-per-line': ['error', 'initializations'],
+
+      // Don't require naming all non-arrow functions. Modern browsers can usually
+      // detect the correct name if it's assigned to a variable or property.
+      'func-names': 'off',
+
+      // Don't require instance methods to use `this` (the rule name is confusing)
+      'class-methods-use-this': 'off',
+
+      // Allow console.log and similar
+      'no-console': 'off',
```

Comment:
Can you give an example of needing to commit console.log?

Reply:
> Can you give an example of needing to commit console.log?

That was in the old file; this was just converted syntax/filename.

But on that note, I could see dropping this option.

Reply:
This was just a migration of one App's ESLint Config. That's likely better to remove.